cli/src/commands/squash.rs

```diff
@@ -97,8 +97,8 @@ pub(crate) struct SquashArgs {
     )]
     into: Option<RevisionArg>,
 
-    /// The revision(s) to use as parent for the new commit (can be repeated
-    /// to create a merge commit)
+    /// (Experimental) The revision(s) to use as parent for the new commit (can
+    /// be repeated to create a merge commit)
```

Comment:
Should we even mark it as hidden so clap doesn't mention it in the help text? That would make sense if the idea is mostly hide the feature while we think more about the UX. I think marking it experimental as you've done here makes more sense if the idea is to collect feedback from users. I'm not sure which we want.

Reply:
I think marking them as experimental is good enough. I would want people to feel free to try them out. In fact, if I were to guess, a large fraction of the people might find these features natural and be confused about why they are "experimental", while another large fraction will be confused by them and be happy to know they don't have to learn them.

I put some more (inconclusive) thoughts in https://github.com/jj-vcs/jj/pull/7162#issuecomment-3226508640.